runtime/doc/lsp.txt

```diff
@@ -71,12 +71,14 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
-                                          *grr* *gra* *grn* *gri* *i_CTRL-S*
+                                 *grr* *gra* *grn* *gri* *gO* *i_CTRL-S*
```

Comment:
The `*gO*` tag should be removed here. Keep the gO mention  below, but also mention the LSP default at the existing `*gO*` tag.

And then check that CI passes, and fixe any problems if not.

Reply:
hey, can you please elaborate what you mean by:

> but also mention the LSP default at the existing *gO* tag.

Wouldn't this line already do that:

```
 "gO" is mapped in Normal mode to |vim.lsp.buf.document_symbol()|
```

I removed the `*gO*` tag

Reply:
I'm confused why removing the tag was necessary though